assets/js/modules/idea-hub/components/dashboard/DashboardCTA.js

```diff
@@ -90,14 +93,6 @@ function DashboardCTA( { Widget, WidgetNull } ) {
 		}
 	}, [ activateModule, navigateTo, setInternalServerError ] );
 
-	const onLinkClick = useCallback( () => {
-		trackEvent(
-			IDEA_HUB_GA_CATEGORY_WIDGET,
-			'click_outgoing_link',
-			'idea_hub_learn_more'
-		);
```

Comment:
Why was this removed? I don't see it mentioned in the ACs/IB, and it seems like something worth tracking—to see if users are interested in this new feature.